src/backend/distributed/operations/repair_shards.c

```diff
@@ -421,6 +423,48 @@ EnsureEnoughDiskSpaceForShardMove(List *colocatedShardList,
 }
 
 
+/*
+ * EnsureTargetNodeIsSafeToMove throws error if he target node is not eligable
+ * for moving shards.
+ */
+void
+EnsureTargetNodeIsSafeToMove(const char *targetNodeName, int targetNodePort)
```

Comment:
`Ensure` sounds like after this method is completed, the target node will be safe for the move, I know that we use this naming style in the codebase but I find `ErrorIfTargetNodeIsNotSafeToMove` more as I think it is more explicit.